src/routers/handlers/individual-psc-list/individualPscListHandler.ts

```diff
@@ -64,7 +68,8 @@ export class IndividualPscListHandler extends GenericHandler<IndividualPscListVi
             confirmationStatementDate,
             dsrEmailAddress,
             dsrPhoneNumber,
-            pscDetails: this.getViewPscDetails(individualPscList, lang),
+            pscDetails,
```

Comment:
don't we need to filter the list of PSC's here to remove the Super secure?
I have one Super and one Individual and both are displayed on the screen.

Reply:
Good spot, i'll add a test to cover this bug